nitrates/llh_analysis/do_bkg_estimation_wPSs_mp2.py

```diff
@@ -69,7 +70,7 @@ def cli():
         action="store_true",
     )
     parser.add_argument(
```

Comment:
is there any reason to keep this argument?

Reply:
I kept it as an option to use a partial coding image if you wanted to and already had one.